### PR TITLE
[rstmgr] Officially transition to D1

### PR DESCRIPTION
**Pull request by tjaychen**
Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

File: hw/ip/rstmgr/doc/checklist.md
Comment (sjgitty) [inline hw/ip/rstmgr/doc/checklist.md:5]: @tjaychen can you remove this comment?

File: hw/ip/rstmgr/doc/checklist.md
Comment (sjgitty) [inline hw/ip/rstmgr/doc/checklist.md:28]: I think we've decided to skip these two lines for D1/V1

**Review by sjgitty — APPROVED** [review]
LGTM, modulo one comment removal request. Can we get @sriyerg to LGTM on the specification?

**Review by msfschaffner — APPROVED** [review]

File: hw/ip/rstmgr/doc/checklist.md
Comment (tjaychen) [reply hw/ip/rstmgr/doc/checklist.md:5]: o yes will do. Forgot on this one.

File: hw/ip/rstmgr/doc/checklist.md
Comment (tjaychen) [reply hw/ip/rstmgr/doc/checklist.md:23]: sorry when you say skip, do you mean they shouldn't be there at all?
because these checklists were actually generated from the template. I can submit another PR to update the template if that helps.

Comment (sjgitty) [reply hw/ip/rstmgr/doc/checklist.md:23]: Let's leave them as is, tackle it as an issue on the template, then clean up the ones afterward if needed. I wouldn't let this block this submission.

File: hw/ip/rstmgr/data/rstmgr.sv.tpl
Comment (sriyerg) [inline hw/ip/rstmgr/data/rstmgr.sv.tpl:324]: I think it might be better to have a dedicated commit for the milestone updates.

File: hw/ip/rstmgr/doc/checklist.md
Comment (sriyerg) [inline hw/ip/rstmgr/doc/checklist.md:19]: nit: perhaps N.A. to indicate it does not have any memories?

**Review by sriyerg — APPROVED** [review]
LGTM, save for two minor comments.

File: hw/ip/rstmgr/doc/checklist.md
Comment (tjaychen) [reply hw/ip/rstmgr/doc/checklist.md:19]: o yeah i can do that.

**Pull request merged**
